Cython/Compiler/FlowControl.py

```diff
@@ -980,7 +980,7 @@ def mark_forloop_target(self, node):
                 if not entry or entry.is_builtin:
                     if function.name == 'reversed' and len(sequence.args) == 1:
                         sequence = sequence.args[0]
-                    elif function.name == 'enumerate' and len(sequence.args) == 1:
+                    elif function.name == 'enumerate' and len(sequence.args) in {1, 2}:
```

Comment:
No need to employ hashing here.
```suggestion
                    elif function.name == 'enumerate' and len(sequence.args) in (1, 2):
```